src/webgpu/shader/validation/v_0033.spec.ts

```diff
@@ -0,0 +1,72 @@
+export const description = `
+Tests for validation rule v-0033.
```

Comment:
Can we expand this to list what validation rule v-0033 is so we don't have to go lookup in the spec?

Reply:
It isn't actually in the spec yet so I didn't know what to write here, but +1